subsys/logging/log_core.c

```diff
@@ -1338,12 +1382,24 @@ static void log_process_thread_func(void *dummy1, void *dummy2, void *dummy3)
 {
 	__ASSERT_NO_MSG(log_backend_count_get() > 0);
 
-	log_init();
+	uint32_t activate_mask = z_log_init(false, false);
+	k_timeout_t timeout = K_MSEC(50); /* Arbitrary value */
```

Comment:
Should this timeout lengthen gradually? In case some backends have problem starting...

Reply:
it was introduced in another PR. We can consider it but maybe not in this PR.